WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailToolbar.swift

```diff
@@ -32,16 +32,28 @@ class ReaderDetailToolbar: UIView, NibLoadable {
         prepareActionButtonsForVoiceOver()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if #available(iOS 13.0, *) {
+            if traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection) {
+                configureActionButtons()
+            }
+        }
+
+        configureButtonTitles()
+    }
+
     func configure(for post: ReaderPost, in viewController: UIViewController) {
         self.post = post
         self.viewController = viewController
 
         likeCountObserver = post.observe(\.likeCount, options: .new) { [weak self] _, _ in
-            self?.configureLikeActionButton(true)
+            self?.configureButtonTitles()
```

Comment:
Should we use an observer, wouldn't it make more sense to keep track of the count in the `didTapLike` `toggleLiked` success/failure methods.

Reply:
I guess we might want to keep that because if that changes anywhere else, it will be reflected here. Are you ok with it?

Reply:
Yeah, works for me. Was just an observation.